packages/commonwealth/client/scripts/views/pages/Community/Members/index.types.ts

```diff
@@ -1,4 +1,4 @@
-export type GroupCategory = 'All' | 'In group' | 'Not in group';
+export type GroupCategory = 'All groups' | 'In group' | 'Not in group';
```

Comment:
again I think we should do 'all' instead of 'all groups' or change labels when we change tabs